mqtt:
  host: a38jxci9chk5ry.iot.eu-west-1.amazonaws.com
  rootCAPath: /home/robert/catkin_ws/src/keys_iot/root-CA.crt
  certificatePath: /home/robert/catkin_ws/src/keys_iot/ROSIRELAND.certificate.pem
  privateKeyPath: /home/robert/catkin_ws/src/keys_iot/ROSIRELAN.private.key
  clientId: ROSTEST

bridge:
  # payload with the String format
  - msg_type: std_msgs.msg:Float64
    topic_from: measuretime/timestamp
    topic_to: timestamptopic
  # payload with the Twist format
  - msg_type: geometry_msgs.msg:Twist
    topic_from: aws_to_device/turtlesim
    topic_to: /turtle1/cmd_vel
